tests/cli/test_ci.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
```

Comment:
This isn't needed I think. So is line 7 above. Probably copy&paste error.

Reply:
You are right, I removed it and squashed to the first commit.